distribution/lib/Standard/Database/0.0.0-dev/src/Internal/DB_Table_Implementation.enso

```diff
@@ -215,11 +214,11 @@ type DB_Table_Implementation
 
                     filter_condition = Filter_Condition.resolve_auto_scoped filter
                     case filter_condition of
+                        Filter_Condition.Predicate _ ->
```

Comment:
- Using `Filter_Condition.Predicate _` instead of `_:Function`
- that looks like a straightforward rewrite